content/docs/language.md

```diff
@@ -1412,16 +1410,16 @@ a type. Make sure the type is accurate as the compiler
 cannot check these signatures for correctness:
 
 ```ante
-extern puts: C.String -> C.Int
+extern puts: Ptr char -> i32
 ```
 
 You can also use extern with a block of declarations:
 
 ```ante
 extern
-    exit: C.Int -> never_returns
+    exit: i32 -> never_returns
     malloc: usz -> Ptr a
-    printf: C.String -> ... -> C.Int
```

Comment:
This is another point where the website has what I'd think the ideal design of what these type signatures would be rather than the current design without a C module. I think we should design such a module eventually but until it is more concrete I'm fine with changing this to how it is defined today.

Good catch on the removing the printf declaration as well.